<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsDialog</class>
 <widget class="QDialog" name="SettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>534</width>
    <height>782</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Todour Settings</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAsNeeded</enum>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>500</width>
        <height>774</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Directory</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit"/>
         </item>
         <item>
          <widget class="QPushButton" name="btnDirBrowse">
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Inactive</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_2">
           <property name="toolTip">
            <string>Keywords that mark an item as inactive. Seperated by ';'</string>
           </property>
           <property name="statusTip">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QGroupBox" name="groupBox_2">
           <property name="title">
            <string>Active Font</string>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QLineEdit" name="le_activeFont">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="text">
               <string>Active</string>
              </property>
              <property name="readOnly">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pb_fontDlgActive">
              <property name="text">
               <string>Font</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pb_colorDlgActive">
              <property name="text">
               <string>Color</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox">
           <property name="title">
            <string>Inactive Font</string>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_6">
            <item>
             <widget class="QLineEdit" name="le_InactiveFont">
              <property name="text">
               <string>Inactive</string>
              </property>
              <property name="readOnly">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pb_fontDlgInactive">
              <property name="text">
               <string>Font</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pb_colorDlgInactive">
              <property name="text">
               <string>Color</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
      <item>
        <widget class="QGroupBox" name="groupBox_4">
         <property name="title">
          <string/>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_9">
          <item>
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Font size&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="sb_fontSize"/>
          </item>
         </layout>
        </widget>
       </item>

       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QGroupBox" name="groupBox_3">
           <property name="title">
            <string>Due </string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_1">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_15">
               <item>
                <widget class="QCheckBox" name="cb_due">
                 <property name="text">
                  <string>Enable due:</string>
                 </property>
              </widget>
             </item>
             <item>
                <widget class="QLabel" name="label_3">
               <property name="text">
                <string>Warning:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="sb_due_warning"/>
             </item>
             <item>
              <widget class="QPushButton" name="pb_warningColor">
               <property name="autoFillBackground">
               <bool>true</bool>
              </property>
              <property name="text">
               <string>Warning Color</string>
              </property>
              <property name="flat">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pb_lateColor">
              <property name="autoFillBackground">
               <bool>true</bool>
              </property>
              <property name="text">
               <string>Late Color</string>
              </property>
              <property name="flat">
               <bool>true</bool>
              </property>
             </widget>
            </item>            
           </layout>
           </item>
            <item>
             <widget class="QCheckBox" name="cb_due_as_threshold">
             <property name="text">
                <string>Treat due as threshold</string>
             </property>
             </widget>
            </item>
           </layout>       
          </widget>
         </item>
        </layout>
       </item>

       <item>
        <widget class="QCheckBox" name="cb_autorefresh">
         <property name="text">
          <string>Refresh automatically when files has changed</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_12">
         <item>
          <widget class="QCheckBox" name="cb_showdates">
           <property name="text">
            <string>Show dates</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="cb_dates">
           <property name="text">
            <string>Add dates to lines</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_13">
         <item>
          <widget class="QCheckBox" name="cb_threshold_inactive">
           <property name="text">
            <string>Treat threshold as inactive</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QCheckBox" name="cb_liveSearch">
         <property name="text">
          <string>Live search</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="cb_hotKey">
         <property name="text">
          <string>Ctrl+Alt+t as Todo hotkey</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="cb_deletedfile">
         <property name="toolTip">
          <string>Save deleted items in deleted.txt</string>
         </property>
         <property name="text">
          <string>Track delete</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="cb_threshold">
         <property name="text">
          <string>Respect threshold dates (t:xxxx-xx-xx)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="cb_threshold_label">
         <property name="text">
          <string>Respect threshold contexts and projecs (t:+project or t:@context)</string>
         </property>
        </widget>
       </item>
        <item>
        <widget class="QCheckBox" name="cb_tray_icon">
         <property name="text">
          <string>Enable tray</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="cb_CheckUpdates">
         <property name="text">
          <string>Check for updates (ca once per week and only at startup. No information is sent)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="cb_removeDoublets">
         <property name="text">
          <string>Remove Duplicate Lines</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_11">
         <item>
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>Default deadline/threshold to add when missing</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="cb_default_threshold">
           <property name="toolTip">
            <string>What to add to a rec: line without any due: or t:</string>
           </property>
           <item>
            <property name="text">
             <string>due:</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>t:</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_10">
         <item>
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>Symbol for &quot;NOT&quot; in search</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="search_not_char">
           <property name="maxLength">
            <number>1</number>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>

       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_11">
         <item>
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>Default priority</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="default_priority">
           <property name="maxLength">
            <number>1</number>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>


    <item>
        <layout class="QHBoxLayout" name="horizontalLayout_16">
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>

       <item>
       <layout class="QHBoxLayout" name="horizontalLayout_14">
         <item>
          <widget class="QGroupBox" name="groupBox_5">
           <property name="title">
            <string>Business days </string>
           </property>
            <layout class="QHBoxLayout" name="horizontalLayout_14">
                   <item>
                   <widget class="QCheckBox" name="cb_monday">
                    <property name="text">
                     <string>Mon</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QCheckBox" name="cb_tuesday">
                    <property name="text">
                     <string>Tue</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QCheckBox" name="cb_wednesday">
                    <property name="text">
                     <string>Wed</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QCheckBox" name="cb_thursday">
                    <property name="text">
                     <string>Thu</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QCheckBox" name="cb_friday">
                    <property name="text">
                     <string>Fri</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QCheckBox" name="cb_saturday">
                    <property name="text">
                     <string>Sat</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QCheckBox" name="cb_sunday">
                    <property name="text">
                     <string>Sun</string>
                    </property>
                   </widget>
                  </item>
                  
               </layout>
              </widget>
             </item>
            </layout>
         </item>           
                  
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_8">
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Priorities at checked</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comb_priorities">
           <property name="currentText">
            <string>Remove</string>
           </property>
           <item>
            <property name="text">
             <string>Remove</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Keep (move to after date) </string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Add pri: tag</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
